packages/zowe-explorer/src/trees/shared/SharedInit.ts

```diff
@@ -394,7 +394,13 @@ export class SharedInit {
             (f) => f.uri.scheme === ZoweScheme.DS || f.uri.scheme === ZoweScheme.USS
         );
         for (const folder of newWorkspaces) {
-            await (folder.uri.scheme === ZoweScheme.DS ? DatasetFSProvider.instance : UssFSProvider.instance).remoteLookupForResource(folder.uri);
+            try {
+                await (folder.uri.scheme === ZoweScheme.DS ? DatasetFSProvider.instance : UssFSProvider.instance).remoteLookupForResource(folder.uri);
+            } catch (err) {
+                if (err instanceof Error) {
+                    ZoweLogger.error(err.toString());
```

Comment:
dumb question:
What else can be thrown here, if not an Error 

And if there is something other than an error, should we at least stringify that object and still add a log entry for it? 😅

Reply:
_Not_ a dumb question - it's mostly there to get the intellisense for the error object and the `toString` function.
It probably is redundant to have that since any error from `remoteLookupForResource` will satisfy the `instanceof Error` check 😅

Reply:
Sure! Once the error correlator is available, we can also log the correlated error, as well as giving users the option to fetch those resources again.